rhobs/test/import/go.mod

```diff
@@ -0,0 +1,114 @@
+module github.com/rhobs/obo-prometheus-operator/rhobs/test/import
+
+go 1.20
+
+require github.com/rhobs/obo-prometheus-operator/pkg/apis/monitoring v0.64.0-rhobs2
+
+require github.com/rhobs/obo-prometheus-operator/pkg/client v0.64.0-rhobs1 // indirect
```

Comment:
Should be require - github.com/rhobs/obo-prometheus-operator instead?

Reply:
this should have been `pkg/client v0.64.0-rhobs2` not `rhobs1`.

Reply:
updated